templates/config/deployment.yaml

```
@@ -15,7 +15,7 @@ metadata:
     {{- end -}}
   {{- end }}
 spec:
-  revisionHistoryLimit: 3
+  revisionHistoryLimit: {{ .Values.geoserver.config.revisionHistoryLimit | default 3 }}
```

Do you relay want different values per deployment?
In my helm chart who has more than one deployment, I steel use 
```suggestion
  revisionHistoryLimit: {{ .Values.revisionHistoryLimit | default 3 }}
```
As you want :-)

---

mmh, I'll put that in global, fair point